shared/js/ui/templates/shared/render-broken-site-href.es6.js

```diff
@@ -1,3 +1,14 @@
-module.exports = function (browser, url) {
-  return `mailto:extension-brokensites@duckduckgo.com?subject=${browser}%20Extension%20Broken%20Site%20Report&body=Help%20us%20improve%20by%20sharing%20a%20little%20info%20about%20the%20issue%20you%27ve%20encountered%2E%0A%0A1%2E%20Which%20website%20is%20broken%3F%20%28copy%20and%20paste%20the%20URL%29%0A%0A2%2E%20Describe%20the%20issue%2E%20%28What%27s%20breaking%20on%20the%20page%3F%20Attach%20a%20screenshot%20if%20possible%29%0A%0A----URL%20is%20${encodeURIComponent(url)}`
+const emailFeedbackFooter = require('./email-feedback-footer.es6.js')
+
+module.exports = function (browserInfo, url) {
+  let subject = `${browserInfo.browser} Extension Broken Site Report`
+  let body = `Help us improve by sharing a little info about the issue you've encountered.
+
+1. Which website is broken? (copy and paste the URL)
+
+2. Describe the issue. (What's breaking on the page? Attach a screenshot if possible)
+
+${emailFeedbackFooter(browserInfo, url)}`
+
+  return `mailto:extension-brokensites@duckduckgo.com?subject=${encodeURIComponent(subject)}&body=${encodeURIComponent(body)}`
```

Comment:
Encoding this on the fly so I didn't have to URL decode and encode again as I was adding the footer text. See above re: laziness.

Reply:
thanks for fixing this. I hadn't seen this when I commented above. I assume you tested this and it works as expected, @andrey-p , @MariagraziaAlastra  ?

Reply:
Yep